src/heif.h

```diff
@@ -273,6 +273,36 @@ void heif_context_free(struct heif_context*);
 
 struct heif_reading_options;
 
+enum heif_reader_offset {
+  heif_seek_start,
+  heif_seek_current,
+  heif_seek_end
```

Comment:
We probably can remove heif_seek_end, because I do not know any case where we would need this.
Omitting this could make the reader easier to write.

Reply:
This is used in `BitstreamRange::skip_to_end_of_file()`.

Reply:
Ok, I see. But skip_to_end_of_file() is only used if a box cannot be parsed because its type is unknown and the box length is "until end of file". Hence, we will find another way to handle this.